noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -894,10 +1363,15 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST8">
         <title>Files within leaf classes</title>
+        <description>
+            Provides information about the number of Files per Series and
```

Comment:
`Provides information about the number of files per class and series, and tests whether any file refers to a class which has other classes as children (i.e. the file has a class sibling)`